src/login/translation/messageIndex.js

```diff
@@ -46,15 +46,20 @@ export const unformattedMessages = defineMessages({
     defaultMessage: "Code is formatted as five groups of characters and numbers, separated by hyphens",
     description: "Help text for email confirmation code",
   },
-  "emails.invalid_code": {
-    id: "emails.invalid_code",
+  "modal.long_code_invalid_format": {
+    id: "modal.long_code_invalid_format",
     defaultMessage: `Invalid confirmation code`,
     description: "error text for invalid confirmation code"
   },
-  "mobile.confirm_code_wrong_length": {
-    id: "mobile.confirm_code_wrong_length",
-    defaultMessage: `Code must be 10-12 characters long, only letters and numbers allowed`,
-    description: "Error message for wrong length",
+  "modal.short_code_invalid_format": {
+    id: "modal.short_code_invalid_format",
+    defaultMessage: `The code is 10 characters long`,
```

Comment:
I really like how you have simplified the logic for validation and the messaging! its really user friendly! 

However, I need some help understanding this one, because it tells me that the code is 10 characters, but it is 9 (see screenshot). It's also in orange so I assume something is wrong, but what I am not sure.
 
<img width="300" alt="Screenshot 2020-10-07 at 10 10 40" src="https://user-images.githubusercontent.com/30963614/95306201-60a58700-0887-11eb-8ddd-6db12e687443.png">

Reply:
I think it was me who convinced Eunju to use this phrase. I agree that it didn't turn out very well (I needed to see it in the screenshot to realise that), but the thing is that we want to convey two things in a short and simple way:

1. The code actually _is_ 10 characters long, because we made it and put it in the letter/SMS.

2. Since the currently entered code is not 10 characters long, the user probably haven't found the right code - or entered it incorrectly.

Maybe just say "Incorrect code" without trying to convey to the user that it is either the length that is clearly wrong, or an invalid character was entered?

Reply:
One option is to add a help text next to the label, "the code is 10 characters long" and change the error message to "incorrect code"

Reply:
I'm not sure it would actually be a helpful help text unless we start checking length separately from the format. If I enter "räksmörgås" (10 characters), "Incorrect code" is correct, but any kind of reference to the length and the number 10 would perhaps only be misleading.